sdks/java/core/src/main/resources/org/apache/beam/sdk/sdk.properties

```diff
@@ -18,3 +18,5 @@
 
 version=@pom.version@
 sdk_version=@pom.sdk_version@
+docker_image_default_repo_root=@pom.docker_image_default_repo_root@
+docker_image_default_repo_prefix=@docker_image_default_repo_prefix@
```

Comment:
```suggestion
docker_image_default_repo_prefix=@pom.docker_image_default_repo_prefix@
```

Reply:
(I don't know if the `pom` part is significant, but some Dataflow Runner code assumes it's there so it's probably best to keep the pattern here.)

Reply:
Yeah, its important. It is used as the ReplaceToken in the build.gradle. Its late here. zzz